CRM/Report/Form/Contribute/Lybunt.php

```diff
@@ -315,8 +312,17 @@ public function where() {
       $this->_where .= " AND {$this->_aclWhere} ";
     }
     CRM_Financial_BAO_FinancialType::getAvailableFinancialTypes($financialTypes);
-    $this->_where .= " AND {$this->_aliases['civicrm_contribution']}.financial_type_id IN (" . implode(',' , array_keys($financialTypes)) . ")";
-    $this->_where .= " AND {$this->_aliases['civicrm_line_item']}.financial_type_id IN (" . implode(',' , array_keys($financialTypes)) . ")";
+    
```

Comment:
I think this whole code should go at line 372 in postprocess function after $this->groupBy().